### DISCUSSION TITLE
Is there a way to run a script after cmake configure?

### DISCUSSION THREAD

**Discussion by juxeii** [microsoft/vscode-cmake-tools#2507] [Q&A] [answered]
I need to run a bash script after a cmake configure has finished.
I want this bash script to be run automatically after each cmake configure.
What would be a simple way of doing so?

**Comment by bobbrow** [comment, marked as answer]
There isn't currently a way to do this, but we are planning to add "configure" to the list of commands supported by our task provider in the 1.11 release.  You could then write a task that runs your script and have it depend on a "configure" task running first.  #2452
